src/main/java/sirius/biz/jobs/batch/file/LineBasedImportJob.java

```diff
@@ -73,9 +73,14 @@ protected void executeForStream(String filename, InputStream in) throws Exceptio
         });
     }
 
+    @Override
+    protected boolean canHandleFileExtension(String fileExtension) {
+        return "xls".equals(fileExtension) || "xlsx".equals(fileExtension) || "csv".equals(fileExtension);
+    }
+
     @Override
     public void handleRow(int index, Values row) {
-        if (aliases == null) {
+        if (index == 1) {
```

Comment:
explain the change in a separate commit!!

Reply:
and also reset the aliases before every run, there is no guarantee that eacht file uses the same format